File: Data/ViewModels/Note.cs
﻿using System;
using System.Collections.Generic;

namespace Nebula.Data.ViewModels
{
    /// <summary>
    /// Nota de Inventario.
    /// </summary>
    public class Note
    {
        public int ContactId { get; set; }
        public Guid WarehouseId { get; set; }
        public string NoteType { get; set; }
        public int Motivo { get; set; }
        public DateTime StartDate { get; set; }
        public string Remark { get; set; }
        public List<ItemNote> ItemNotes { get; set; }
    }
}

File: Angular/src/app/inventory/interfaces/note.ts
import {ItemNote} from './item-note';

export class Note {
  constructor(
    public contactId: number = 0,
    public warehouseId: number = 0,
    public noteType: string = '',
    public motivo: number | string = 0,
    public startDate: any = null,
    public itemNotes: Array<ItemNote> = new Array<ItemNote>()) {
  }
}
